apps/coscrad-frontend/src/components/timeline/timeline.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, Slider } from '@mui/material';
+
+export interface TimelineMark {
+    value: number;
+    label: React.ReactNode;
+}
+
+interface TimelineProps {
+    defaultValue: number;
+    step: number;
+    min: number;
+    max: number;
+    marks: TimelineMark[];
+    name: string;
+}
+
+export const Timeline = ({ marks, defaultValue, step, min, max, name }: TimelineProps) => {
```

Comment:
So here, if I switch out the MUI slider for a custom timeline component, this API would change